src/section-function-basics.ptx

```diff
@@ -1158,6 +1158,27 @@
         </example> -->
 
     <exercises>
+        <exercisegroup cols="2">
+            <title>Review and Warmup</title>  
+            <exercise>
+                <webwork source="BasicAlgebra/EvaluateExpressions/evaluate80.pg" />
+            </exercise>
+            <exercise>
+                <webwork source="BasicAlgebra/EvaluateExpressions/evaluate80.pg" />
+            </exercise>
+            <exercise>
+                <webwork source="BasicAlgebra/EvaluateExpressions/evaluate130.pg" />
+            </exercise>
+            <exercise>
+                <webwork source="BasicAlgebra/EvaluateExpressions/evaluate130.pg" />
+            </exercise>
+            <exercise>
+                <webwork source="ORCCA-201802/4-Graphing_Lines/4_1-Cartesian_Coordinates/4_1_1_1.pg" />
```

Comment:
The source for this one is a file in ORCCA-201802/...  We do not want to do this. That makes this edition rely on a problem that was in the source code of a previous edition. For one thing, if another school picked up ORCCA they would not have an ORCCA-201802 folder to reference on their WW server. This problem's source code (or an updated version of it) is going to be in the file for section 4.1 (section-caretsian-coordinates.ptx). So the thing to do is copy that chunk of code and paste it here.

Could you do that and update this PR? If you don't get to it by later tonight, no worries. I will then accept this as it is and immediately paste over this one and any others like this.